dbt/adapters/base/impl.py

```diff
@@ -1535,6 +1535,9 @@ def process_parsed_constraint(
         parsed_constraint: Union[ColumnLevelConstraint, ModelLevelConstraint],
         render_func,
     ) -> Optional[str]:
+        # skip checking enforcement if this is a 'custom' constraint
```

Comment:
Put another way, are we saying that custom constraints are always supported and always enforced? Or do we still need to check that?

Reply:
I just read the issue. So it seems as though custom constraints are always supported and _never_ enforced. But because of this, we wouldn't need to warn on not enforced as the user knows what they're signing up for. This makes sense then.